README.md

```diff
@@ -1,15 +1,13 @@
 # jscs-config-seegno
-
 Seegno-flavored JSCS config.
 
 ## Installation
 
 ```sh
-$ npm install jscs jscs-config-seegno --save-dev
+$ npm install jscs jscs-preset-seegno should --save-dev
```

Comment:
I would move the preset rename to another PR so we can rename the repo + package too.